app/views/observations/index.html.haml

```diff
@@ -147,9 +146,33 @@
                     = "{{ p.placeTypeLabel( ) }}"
                 %span{ "ng-show": "!nearbyPlaces.community || nearbyPlaces.community.length == 0" }
                   {{ shared.t("no_places_available") }}
-            %button.button.btn-default.reload{ "ng-click": "filterByBounds( )", "ng-show": "viewing('observations', 'map') && !hideRedoSearch" }
-              %span.glyphicon.glyphicon-repeat{ "aria-hidden": "true" }
-              {{ shared.t('redo_search_in_map') }}
+            %span.map-shape-searches
+              %div.shape-buttons
```

Comment:
I grouped the buttons in a div so they'd wrap together on small screens. But perhaps expanding it to include the Fit to Map button?